### PR TITLE
Update UI with drawer style

### LINKED ISSUE #6: 顯示在每一頁
不只購物車頁面要顯示。最好可以做成常駐在頁面的可隱藏彈出式功能。

### PR DESCRIPTION
**Pull request by focaaby**
- Update the UI as the following demo screenshot:
  - 1_hidden
  - 2_open
- Fixed #6 #5

### CODE REVIEW

File: index.user.js [hunk 1]
```diff
@@ -119,10 +119,39 @@
...
-    const d = document.querySelector('#main > div.main__content > div > div.trolley--wrapper > div > div > div.full-trolley--grid > div.full-trolley--content > div:nth-child(3) > div')
-    d.append(toolsNode)
+
+    const toolsHTML = `
+        <div style="
+                position: fixed;
+                top: 60%;
```
Comment (aldy120) [inline index.user.js]: `top: 60%` 有機會在解析度低的畫面無法完全顯示
Screen Shot 2022-02-28 at 20 18 48

Comment (aldy120) [reply index.user.js]: 似乎可以改
```css
z-index: 100;
top: 0
```

**Review by aldy120 — CHANGES_REQUESTED** [review]

File: index.user.js (hunk 1 above)
Comment (focaaby) [reply index.user.js]: 本來是想說不要擋到上面的選單，不過也可以改成做側欄伸縮

**Comment by focaaby** [comment]
Modify from `top:60%` to `top: 0`.
image

**Comment by aldy120** [comment]
Resolve #5

**Pull request merged**